apps/protocol-frontend/src/components/DaoDashboardShell.tsx

```diff
@@ -23,81 +25,45 @@ const DaoDashboardShell = ({ daoName }: DaoDashboardShellProps) => {
         background="white"
         boxShadow="sm"
         borderRadius={{ base: 'none', md: 'lg' }}
-      >
-        <Flex direction="column" gap={4}>
-          <Flex direction={{ base: 'column' }} gap={2}>
-            <Flex
-              direction={{ base: 'column', lg: 'row' }}
-              gap={8}
-              justifyContent="space-apart"
-            >
-              {/* <Flex flexBasis="50%"> */}
-              {/* DatePicker will go here */}
-              {/* </Flex> */}
-            </Flex>
-            <Flex direction="column" gap={2}>
-              <Heading
-                as="h3"
-                fontSize="lg"
-                color="gray.800"
-                fontWeight="normal"
-              >
-                Contributions By Date
-              </Heading>
-            </Flex>
-            <Flex direction="column" gap={2}>
-              <Heading
-                as="h3"
-                fontSize="lg"
-                color="gray.800"
-                fontWeight="normal"
-              >
-                Contributions By Member
-              </Heading>
-            </Flex>
-            <Flex direction="column" gap={2}>
-              <Heading
-                as="h3"
-                fontSize="lg"
-                color="gray.800"
-                fontWeight="normal"
-              >
-                Contributions By Type
-              </Heading>
-            </Flex>
-            <Flex direction="column" gap={2}>
-              <Heading
-                as="h3"
-                fontSize="lg"
-                color="gray.800"
-                fontWeight="normal"
-              >
-                Recent Contributions
-              </Heading>
-            </Flex>
-            <Flex direction="column" gap={2}>
-              <Heading
-                as="h3"
-                fontSize="lg"
-                color="gray.800"
-                fontWeight="normal"
-              >
-                Active Members
-              </Heading>
-            </Flex>
-            <Flex direction="column" gap={2}>
-              <Heading
-                as="h3"
-                fontSize="lg"
-                color="gray.800"
-                fontWeight="normal"
-              >
-                Contributors This Month
-              </Heading>
-            </Flex>
+      > */}
+      <Flex direction="column" gap={4}>
+        <Flex direction={{ base: 'column' }} gap={2}>
+          <Flex
+            direction={{ base: 'column', lg: 'row' }}
+            gap={8}
+            justifyContent="space-apart"
+          ></Flex>
+          <Flex direction="column" gap={2}>
+            <Heading as="h3" fontSize="lg" color="gray.800" fontWeight="normal">
```

Comment:
Could fontSize, color and fontWeight be passed on the top level div so the props are repeated

Reply:
Great idea! We should do that more often generally. We do it at some parts

Reply:
I made this change, but it had some side effects -- as we tweak the actual layout for this page I'll optimize this